tests/testthat/test-dummy.R

```diff
@@ -354,6 +354,34 @@ test_that("throws an informative error for single level", {
   )
 })
 
+test_that("sparse = TRUE works", {
+  rec <- recipe(~ ., data = tibble(x = c(NA, letters)))
+
+  suppressWarnings({
+    dense <- rec %>% step_dummy(x, sparse = FALSE) %>% prep() %>% bake(NULL)
+    dense <- purrr::map(dense, as.integer) %>% tibble::new_tibble()
```

Comment:
There's a portion in the `step_dummy()` documentation that reads "The columns returned by this step are doubles (not integers)"—let's clarify that that's only the case when `sparse = FALSE`.

Reply:
done in [`6e23367` (#1392)](https://github.com/tidymodels/recipes/pull/1392/commits/6e23367e089c82726b89791a205ffc28a5bd9749)